playbooks/base/pre.yaml

```diff
@@ -18,6 +18,12 @@
 - hosts: all:!appliance*
   tasks:
     - name: Run configure-swap role
+      # configuring swap can fail for multiple reasons and should not be
+      # enabled unless requested
+      # https://bugs.launchpad.net/ubuntu/+source/util-linux/+bug/1788321
+      # https://unix.stackexchange.com/questions/294600/i-cant-enable-swap-space-on-centos-7
+      # https://bugzilla.redhat.com/show_bug.cgi?id=1129205
+      when: configure_wrap | default(false)
```

Comment:
`configure_wrap` -> `configure_swap` :-)

- I would call it `enable_swap` to avoid the name confusion with the role name
- Since it's the default today for all our jobs, the value has to default to `True`, to avoid any disruption.